config/local-development/tilt/replace-image.yaml

```diff
@@ -1,4 +1,4 @@
 - op: replace
-  path: /spec/template/spec/containers/1/image
+  path: /spec/template/spec/containers/0/image
   value:
-    quay.io/raffaelespazzoli/vault-config-operator:latest
```

Comment:
does this still not work? I cannot accept this. After the new rebase this should have been fixed for everyone.

Reply:
fix this in separate PR